### PR TITLE
feat(ActionModal): added support for custom header component

### PR DESCRIPTION
**Pull request by ritz078**

### CODE REVIEW

File: storybook/stories/actionModal.story.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                show: false
+              })
+            }
+            style={{
+              children: {
+                paddingTop: 0
+              }
+            }}
+            header={
+              <View>
```
Comment (l3satwik) [inline storybook/stories/actionModal.story.tsx]: Looks like an extra `<View>...</View>`

File: storybook/stories/actionModal.story.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                <View style={styles.container}>
+                  <Text>Add a note</Text>
+
+                  <Text color={colors.violet.base} onPress={() => {}}>
```
Comment (speedcuber911) [inline storybook/stories/actionModal.story.tsx]: Why not noop() like everywhere else?

File: storybook/stories/actionModal.story.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                </View>
+              </View>
+            }
+          >
+            <TextInput
+              placeholder="Enter Note"
+              multiline
+              numberOfLines={6}
+              style={{
+                textAlignVertical: "top"
```
Comment (speedcuber911) [inline storybook/stories/actionModal.story.tsx]: Inline styles.

File: storybook/stories/actionModal.story.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+          <ActionModal
+            visible={store.show}
+            onClose={() =>
+              setState({
+                show: false
+              })
+            }
+            style={{
+              children: {
+                paddingTop: 0
```
Comment (speedcuber911) [inline storybook/stories/actionModal.story.tsx]: inline styles.

**Pull request merged**